src/web-fls-quiz/Models/Quiz.cs

```diff
@@ -0,0 +1,13 @@
+﻿namespace WebFlsQuiz.Models
+{
+    public class Quiz
+    {
+        public int Id { get; set; }
+
+        public string Name { get; set; }
+
+        public string Greeting { get; set; }
+
+        public QuizOrganizer Organizer { get; set; }
```

Comment:
QuizOrganizer is advised for implementation according to the Interested Party pattern.